cdap-docs/admin-manual/source/security/authorization.rst

```diff
@@ -210,7 +200,9 @@ Programs
    * - Get
      - At least one of *READ, WRITE, EXECUTE,* or *ADMIN*
    * - Resume/Suspend schedule
-     - *EXECUTE*
+     - *EXECUTE* on the program
+   * - Add/Delete/Update schedule
+     - *ADMIN* on the application
```

Comment:
From `Pre-grant and Wildcard Privileges` section on line 349 -
Let's remove the pre-grant part. 

Also move `When CDAP is first started with authorization enabled, no users are granted privileges on any CDAP entities. Without any privileges, CDAP will not be able to create the default namespace. To create the default namespace, grant ADMIN on default namespace to the CDAP master user. The default namespace will get created in several minutes automatically.` to the end of section `Enabling Authorization`

Now the section on line 349 becomes -
### Wildcard Privileges
Wildcard privileges can be used to simplify granting privileges on multiple entities. The following sections provide examples on granting wildcard privileges.